tiledb/sm/c_api/tiledb_serialization.h

```diff
@@ -161,6 +161,25 @@ TILEDB_EXPORT int32_t tiledb_deserialize_array_schema(
     int32_t client_side,
     tiledb_array_schema_t** array_schema) TILEDB_NOEXCEPT;
 
+/**
+ * Deserializes an array creation request from the given buffer.
+ *
+ * @param ctx The TileDB context.
+ * @param buffer Buffer to deserialize from
+ * @param serialization_type Type of serialization to use
+ * @param client_side Allows to specify different behavior depending on who is
+ * serializing, the client (1) or the Cloud server (0). This is sometimes needed
+ * since they are both using the same Core library APIs for serialization.
+ * @param array_schema Will be set to a newly created array schema.
+ * @return `TILEDB_OK` for success and `TILEDB_ERR` for error.
+ */
+TILEDB_EXPORT int32_t tiledb_deserialize_array_create(
```

Comment:
Should we call it `tiledb_handle_array_create` like some other APIs? Returning an array schema does not address the fact that a schema object contains a URI, which does not make much sense conceptually.

Reply:
I pushed a commit with what I had in mind based on my previous comment but feel free to revert if you think otherwise.